examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.c

```diff
@@ -198,6 +208,42 @@ sl_status_t join_callback_handler(sl_wifi_event_t event, char * result, uint32_t
 }
 
 #if SL_ICD_ENABLED
+
+#if SIWX_917
+/******************************************************************
+ * @fn   M4_sleep_wakeup()
+ * @brief
+ *       M4 going to sleep
+ *
+ * @param[in] None
+ * @return
+ *        None
+ *********************************************************************/
+void M4_sleep_wakeup()
```

Comment:
I am not a fan of the function name. It doesn't fit the rest of the apis

Reply:
renamed the API to `sl_wfx_host_si91x_sleep_wakeup()`